src/tests/lib/generator.test.js

```
@@ -0,0 +1,54 @@
+            expect(code).toHaveLength(20)
+            expect(code).toMatch(/^[0-9a-f]+$/)
+        })
+
+        test('should not generate same code even after 5 loops', () => {
+
+            var arr = [1, 2, 3, 4, 5]
+            for(var i = 0; i < 5; i++) {
+                arr[i] = workspaceInviteCodeGenerator()
+            }
+
+            for(var i = 0; i < 5; i++) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)